ARCharts/ARBarChartProtocols.swift

```diff
@@ -53,7 +57,3 @@ public protocol ARBarChartDataSource: class {
                   forSeries series: Int) -> UIColor
```

Comment:
This method should be part of the delegate instead of data source since it has to do with customization and not data.